### ISSUE TITLE
sendMessage error on unlicensed accounts

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2736]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.38.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.38.0...4.38.1) (2022-06-07)

- 🚑 fix #2736 [`#2736`](https://github.com/open-wa/wa-automate-nodejs/issues/2736)
- Release 4.38.1 [`445ab68`](https://github.com/open-wa/wa-automate-nodejs/commit/445ab6888007e828641dc6cf4b75e8499a7626d2)
- 🥚 patches update [`c377734`](https://github.com/open-wa/wa-automate-nodejs/commit/c377734603b88d301c0ed46134379ff4ec5e64eb)
- 🥚 patches update [`3353b87`](https://github.com/open-wa/wa-automate-nodejs/commit/3353b8738940f4bbdaa89817f2a07170e352447d)
- 🥚 patches update [`2d03eec`](https://github.com/open-wa/wa-automate-nodejs/commit/2d03eec4dd54b44fe815bee0e7e3669ad1b52a7c)
- updated types-only package [`8dd117c`](https://github.com/open-wa/wa-automate-nodejs/commit/8dd117ce5109f2bc0b3d001732dbfa92301270bd)
